presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -445,3 +440,9 @@ Statistical Aggregate Functions
 .. function:: var_samp(x) -> double
 
     Returns the sample variance of all input values.
+
+--------------------------
+
+ .. [BenHaimTomTov2010] Yael Ben-Haim and Elad Tom-Tov, "A streaming parallel decision tree algorithm",
+    J. Machine Learning Research 11 (2010), pp. 849--872.
+    Returns the sample variance of all input values.
```

Comment:
Delete this line, which repeats the definition of var_samp

Reply:
Sorry!